howso/client/base.py

```diff
@@ -201,6 +213,33 @@ def get_trainee_session_indices(self, trainee_id, session) -> Union[Index, List[
     def get_trainee_session_training_indices(self, trainee_id, session) -> Union[Index, List[int]]:
         """Get list of all session training indices for a specified session."""
 
+    @abstractmethod
+    def get_hierarchy(self, trainee_id) -> Dict:
+        """Output the hierarchy for a trainee."""
+
+    @abstractmethod
+    def rename(
```

Comment:
So `rename` is just for heirarchy trainees? Feels too generally named.

Reply:
that's what's it is in core and will be once we get rid of the howso layer.  both execute and rename are core-centric methods

Reply:
renaming to rename_subtrainee